core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,29 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    abort-all-workflows-activity {
+      depends-on = [service-unbind]
+      timeout = 1 hour
+    }
+    
+    # This phase is the one responsible for flushing all outstanding write requests to the Database
+    stop-io-activity {
+      depends-on = [service-stop]
+      timeout = 30 seconds
+    }
+  }
 }
 
 system {
   # If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
-  #abort-jobs-on-terminate = false
+  abort-jobs-on-terminate = false
+  
+  # If 'true', a SIGINT will trigger Cromwell to attempt to gracefully shutdown in server mode, in particular clearing up all 
```

Comment:
Sure, but only in the same way that this whole PR is changing behavior. One typically shuts down a long lived server via `kill` which sends a SIGTERM. The point of this exercise is to provide graceful shutdowns for the server. But yes, shutdown procedures should be documented